### PR TITLE
Add Tunlookup App

### PR DESCRIPTION
**Pull request by tun-application**

### CODE REVIEW

**Comment by zeke** [comment]
I would suggest changing the description to something like "Translate selected text from your menu bar", or "Translate selected text with a keystroke", or similar depending on how your app works.

If the app interface is not in English, please also add a `locales` array. See https://github.com/electron/electron-apps/blob/master/contributing.md#locales

**Comment by tun-application** [comment]
Thank for your suggestion, I has changed the description. 
And about language my app, only support english, so `locales ` is unnecessary. pls approve my app.
Thank a lot!

File: apps/tunlookup/tunlookup.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Tunlookup
+description: 'Translate selected text with a keystroke'
+website: 'https://tun-application.github.io/'
+keywords:
+    - Tunlookup
+    - Translate selected text with a keystroke
```
Comment (zeke) [inline apps/tunlookup/tunlookup.yml]: These first two keywords can be removed, as they are the same as the name and description of the app.

**Comment by zeke** [comment]
👍

**Pull request merged**